user_sync/app.py

```diff
@@ -75,6 +75,15 @@ def process_args():
     parser.add_argument('-d', '--remove-list',
                         help='specifies the file containing the list of users to be removed. Users on this list are removeFromOrg\'ed on the Adobe side.',
                         metavar='input_path', dest='remove_list_input_path')
+    parser.add_argument('--delete-nonexistent-users',
+                        help='Causes the user sync tool to delete users that exist on the Adobe side if they are not in the customer side AD. If the account is an Adobe account, it is remove from org\'d, but not deleted.',
+                        action='store_true', dest='delete_nonexistent_users')
+    parser.add_argument('--generate-delete-list',
```

Comment:
ah yeah I added it because as far as I could tell, the --generate-remove-list would only include federated users... --generate-delete-list also includes orphaned managed users. Should I remove this?